README.md

```diff
@@ -355,25 +350,25 @@ In the whole examined sample, it is 331:479 of incorrect:correct (sum 3*270=810)
 In the 'High-confidence' subset, it is also increasing somewhat (13-15-242), but the divide is much sharper between strictly taken 'incorrect' and 'correct'.<br>
-This suggests that it could be worthwhile to examine further whether the higher the reported certainty score for a prediction is, the higher its chance for correctness actually is, as compared to human annotations. While these prelimiary figures look promising, they are produced in a rather simple way, and cannot be taken as evidence for such a trend. 
+This suggests that it could be worthwhile to examine further whether the higher the reported certainty score for a prediction is, the higher its chance for correctness actually is, as compared to human annotations. While these preliminary figures look promising, they are produced in a rather simple way, and cannot be taken as evidence for such a trend. 
 
-**H2: CLIP3 performed significantly better in one or more categories (as compared to other categories).** and
+**H2: CLIP3 performed significantly better in one or more categories (as compared to other categories)**, and
 
-**H3: CLIP3 performed significantly worse in one or more categories (as compared to other categories).** had to be considered beyond the scope of this task. However, hopefully its results will provide useful background for anyone who might decide to examine them in the future (however, NB that the dataset itself is available upon request).
+**H3: CLIP3 performed significantly worse in one or more categories (as compared to other categories)** had to be considered beyond the scope of this task. However, hopefully its results will provide useful background for anyone who might decide to examine them in the future (however, NB that the dataset itself is available upon request).
 
-**H4a: A significant amount of first-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence.** and
+**H4a: A significant amount of first-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence**, and
 
-**H4b: A significant amount of second-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence.** are both supported by this task's analysis, meaning that neither choice given was actually correct when its confidence score was low (Although NB the second-choice caveat as referred to also in HBb right below).
+**H4b: A significant amount of second-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence** are both supported by this task's analysis, meaning that neither choice given was actually correct when its confidence score was low (Although NB the second-choice caveat as referred to also in HBb right below).
```

Comment:
also NB + `HBb` may have been supposed to be `H5b`?

Reply:
🅱️